### PR TITLE
Creative commons changes. Links created as on the IMOS portal

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -49,12 +49,13 @@ environments {
     }
 }
 
+// production uses Chef managed values
```
Comment (danfruehauf) [inline grails-app/conf/Config.groovy]: You can get rid of that comment, it's implied.

**Comment by pmbohm** [comment]
No i cant as both Laurent  and I spent 10 minutes wondering why production was ok when the code wasnt @danfruehauf

File: grails-app/conf/Config.groovy (hunk 1 above)
Comment (danfruehauf) [inline grails-app/conf/Config.groovy]: @pmbohm You commented in the wrong place - the discussion is here.

Again, this is implied, it doesn't matter how much time you and Loz spent. If you think those defaults should be changed - then change them (on a different commit of course). This repository has nothing to do with chef.

**Comment by pmbohm** [comment]
Separate pull request for dev environment variables now created @danfruehauf

**Comment by danfruehauf** [comment]
Rebase against master (after you pull master).

**Comment by pmbohm** [comment]
@danfruehauf  agh better thanks

**Pull request merged**